serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/States.kt

```diff
@@ -476,12 +489,18 @@ class MatrixChatRoom(private val msession: MatrixSession, val room_ids: List<Str
         }
         return this
     }
-    fun sendReply(msg: String, in_reply_to_id: String): MatrixState {
+    fun sendReply(msg: String, in_reply_to_id: String, format: String = ""): MatrixState {
         val in_reply_to = toSharedUiMessageList(msession, username, room_id, 0, in_reply_to_id, 0, true).first.firstOrNull()
+        var formatted_body = msg
+        var body = getUnformattedBody(msg)
         val message = (in_reply_to?.let { event ->
+            val server = msession.server
+            val prev_sender = event.sender
+            val prev_content = event.message
```

Comment:
We should prob expose formatted_message, and have this be copied into the formatted reply